packages/lesswrong/components/posts/PostsPage/BestOfLessWrong/SplashImageEditing.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState, useRef, RefObject } from 'react';
+import { Components, registerComponent } from '@/lib/vulcan-lib/components';
+import { defineStyles, useStyles } from '@/components/hooks/useStyles';
+import { useHover } from '@/components/common/withHover';
+import { Coordinates } from './ImageCropPreview';
+import { useImageContext } from '../ImageContext';
+const styles = defineStyles("SplashImageEditing", (theme: ThemeType) => ({ 
+  rightSectionBelowBottomRow: {
+    position: 'absolute',
+    top: 200,
+    right: 25,
+    display: 'flex',
+    flexDirection: 'row-reverse',
+    paddingLeft: 8,
+    paddingRight: 8,
+    paddingBottom: 8,
+    zIndex: 2,
+    [theme.breakpoints.down('md')]: {
+      display: 'none',
+    },
+  },
+  changeImageBox: {
+    ...theme.typography.commentStyle,
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    padding: '8px',
+    borderRadius: '4px',
+    backgroundColor: theme.palette.panelBackground.reviewGold,
+    color: theme.palette.text.alwaysWhite,
+    cursor: 'pointer',
+    marginBottom: 8,
+    opacity: 0.3,
+    '&:hover': {
+      opacity: 1
+    }
+  },
+  splashContent: {
+    padding: 8,
+    backgroundColor: theme.palette.background.paper,
+  }
+}));
+
+const SplashImageEditing = ({ imgRef, imageFlipped, setImageFlipped, post }: { imgRef: RefObject<HTMLImageElement>, imageFlipped: boolean, setImageFlipped: (imageFlipped: boolean) => void, post: PostsWithNavigation|PostsWithNavigationAndRevision}) => {
+  const classes = useStyles(styles);
+  const { SplashHeaderImageOptions, ImageCropPreview, LWPopper } = Components;
+
+  const { anchorEl, hover, eventHandlers } = useHover();
+
+  const [cropPreviewEnabled, setCropPreviewEnabled] = useState(false);
+
+  const toggleImageFlip = () => setImageFlipped(!imageFlipped);
+
+  const backgroundImgWrapperRef = useRef<HTMLDivElement>(null);
+  const backgroundImgCropPreviewRef = useRef<HTMLDivElement>(null);
```

Comment:
I don't think we need these any more. Seems like they'll always be null, which should cause `setCropPreview` to bail out 100% of the time